app/src/main/res/values/colors.xml

```diff
@@ -29,5 +29,8 @@
     <color name="color_orange_light_white">#FFB185</color>
     <color name="color_orange_light_transparent">#FFE3D7</color>
     <color name="color_gray_mos">#363636</color>
+    <color name="color_blue_light">#B2C9D0</color>
+    <color name="color_green_dark">#339933</color>
```

Comment:
WARNING: [UnusedResources] The resource `R.color.color_green_dark` appears to be unused
 Unused resources make applications larger and slow down builds.